HCCGo/app/js/jobSubmissionCtrl.js

```diff
@@ -8,6 +8,16 @@ jobSubmissionModule.controller('jobSubmissionCtrl', ['$scope', '$log', '$timeout
   var submittedJobsDB = dbService.getSubmittedJobsDB();
   var jobHistoryDB = dbService.getJobHistoryDB();
 
+  // Initialize the ace editor
+  require("ace-builds/src-noconflict/ace");
```

Comment:
Why are we using this rather than using bower?  I don't think it should be using npm for ace, it's a browser javascript.